examples/common/00-prereqs.yaml

```diff
@@ -111,42 +117,9 @@ metadata:
     component: ark
 subjects:
   - kind: ServiceAccount
-    namespace: heptio-ark
+    namespace: heptio-ark-server
     name: ark
 roleRef:
   kind: ClusterRole
   name: cluster-admin
   apiGroup: rbac.authorization.k8s.io
-
----
-apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
why's the RBAC stuff going away?

Reply:
Because our service account has cluster-admin (granted above)